contracts/modules/LoanOpenings/LoanOpenings.sol

```diff
@@ -89,6 +94,79 @@ contract LoanOpenings is State, LoanOpeningsEvents, VaultController, InterestHan
         );
     }
 
+    function getDefaultLoanParams(
```

Comment:
should this be renamed? It seems to return custom loan params as well as default so shouldnt it just be getLoanParams()?

Reply:
fixed